packages/@okta/vuepress-site/docs/reference/api/schemas/index.md

```diff
@@ -2430,10 +2431,11 @@ In case `enum` is used in conjunction with `oneOf`, the set of enumerated values
 
 Okta has also extended [JSON Schema Draft 4](https://tools.ietf.org/html/draft-zyp-json-schema-04) with the following keywords:
 
-| Property      | Description                                     | DataType                                                                  | Nullable | Unique | Readonly |
-| :------------- | :----------------------------------------------- | :------------------------------------------------------------------------- | :--------- | :------ | :-------- |
-| required      | Determines whether the property is required     | Boolean                                                                   | FALSE    | FALSE  | FALSE    |
-| scope         | Determines whether a group attribute can be set at the individual or group level | `SELF`, `NONE`                           | FALSE    | FALSE  | TRUE     |
+| Property                                                   | Description                                                                      | DataType                    | Nullable | Unique | Readonly |
+| :--------------------------------------------------------- | :------------------------------------------------------------------------------- | :-------------------------- | :------- | :----- | :------- |
+| required                                                   | Determines whether the property is required                                      | Boolean                     | FALSE    | FALSE  | FALSE    |
+| scope                                                      | Determines whether a group attribute can be set at the individual or group level | `SELF`, `NONE`              | FALSE    | FALSE  | TRUE     |
+| [unique](#unique-attribute) <ApiLifecycle access="ea" />   | Determines whether the property enforces uniqueness constraint                   | Boolean                     | FALSE    | FALSE  | FALSE    |
```

Comment:
For ease of review:
![Screen Shot 2021-06-30 at 12 24 35 PM](https://user-images.githubusercontent.com/12566503/124019915-7e2ef980-d99e-11eb-8a08-ad96d73c710d.png)